### ISSUE TITLE
Debug info installer uses a deprecated DNF API

### ISSUE THREAD

**Issue by jfilak** [abrt/libreport#438]
The 'logging' function is not a part of DNF API and will be removed in the upcoming DNF release. Please use only officially supported API functions. DNF API documentation is available at https://dnf.readthedocs.org/en/latest/api.html.
The 'logging' function is not a part of DNF API and will be removed in the upcoming DNF release. Please use only officially supported API functions. DNF API documentation is available at https://dnf.readthedocs.org/en/latest/api.html.

**Comment by jfilak** [comment]
https://github.com/abrt/libreport/blob/master/src/client-python/reportclient/dnfdebuginfo.py

**Comment by jfilak** [comment]
```
# cd /var/spool/abrt/ccpp-...
# abrt-action-analyze-ccpp-local
```

**Comment by jfilak** [comment]
http://abrt.readthedocs.io/en/latest/design.html#debuginfo

```
# rm -rf /var/cache/abrt-di/*
```

**Comment by jfilak** [comment]
https://github.com/rhinstaller/lorax/pull/127/commits/404d6a7405f8c5f69ad90cf5a25fd279e4bca3e5

**Comment by jfilak** [comment]
Looks like an older fix for logging (commit 6536b2daef9b6ffb700ea9508aea9be590e6ac7a) started provoking DNF to issues the error message. Thank your the analysis Ales!

**Comment by arhledvink** [comment]
dnf 2.0 will introduce error
LANG=en_US ../abrt-action-install-debuginfo 
Coredump references 4 debuginfo files, 4 of them are not installed
Initializing package manager
Unknown configuration option: enabled_metadata = 1
Unknown configuration option: enabled_metadata = 1
Setting up repositories
